.github/workflows/pr-author-assigne.yml

```
@@ -0,0 +1,111 @@
+            fi
+
+            # Check if PR author is not among assignees
+            if ! echo "$ASSIGNEES" | grep -qw "$PR_AUTHOR"; then
+              echo "PR author is not the assignee, closing..."
+              gh pr close "${{ github.event.pull_request.number }}" --repo "${{ github.repository }}" \
+              --comment "PR is assigned to someone else. Please refrain from opening PRs until the issue is assigned to you."
+              exit 0
+            fi
+
+            echo "PR author is assigned to the issue, all good."
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)